roles/openshift_efk/README.md

```diff
@@ -0,0 +1,9 @@
+
+
+
+ Required vars:
+
+#kibana_hostname: kibana.example.com
+#es_cluster_size: 1
+#master_url: https://localhost:8443
+deployer_secret_vars: kibana.crt=/etc/origin/master/ca.crt kibana.key=/etc/origin/master/ca.key ca.crt=/etc/origin/master/ca.crt ca.key=/etc/origin/master/ca.key
```

Comment:
um its required to provide something in the secret yes. I should just have it default to `noting=/dev/null`